snakemake/executors/google_lifesciences.py

```diff
@@ -580,12 +580,10 @@ def _set_snakefile(self):
         execution into a temporary directory, and the initial Snakefile
         was somewhere else on the system.
         """
-        from snakemake import SNAKEFILE_CHOICES
-
-        for snakefile in SNAKEFILE_CHOICES:
-            if os.path.exists(os.path.join(self.workdir, snakefile)):
-                self.snakefile = snakefile
-                break
+        if os.path.exists(self.workflow.snakefile):
```

Comment:
I think above doc comment is a bit outdated now. Also, I think if self.workflow.snakefile does not exist this should not silently exit. Just put an assertion for existence at the beginning instead of an if.

Reply:
All set.